audit/conn/noop.go

```diff
@@ -0,0 +1,35 @@
+package conn
+
+import (
+	"context"
+
+	"github.com/microsoft/go-otel-audit/audit/msgs"
+)
+
+// Compile check on interface implementation.
+var _ Audit = NoOP{}
+
+// NoOP is a no-op connection to the remote audit server. This implements conn.Audit interface.
+// Use this when you don't want to send audit messages to the remote audit server.
+type NoOP struct{}
```

Comment:
What exactly is the purpose of this? For testing or something?

Reply:
Exactly. It also lets you start building this code in before you have all your jarvis stuff setup.  You can then just switch out for the production one.